examples/js/calendar-widget/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "example-instantsearch-calendar-widget",
-  "version": "1.0.1",
+  "version": "1.0.2",
```

Comment:
Odd, no `fix` was done on this lately.

Reply:
everything which depends on something which has its version changed changes too, so as InstantSearch is updated, this updates too. That's lerna behaviour which can't be changed as far as I can tell

Reply:
as far as I can tell you can't ask it to update the dependency update, but not update a version? but possibly the version number can be removed from those packages, as they aren't depended upon or published

more info here: https://github.com/lerna/lerna/issues/1586